index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
+                                <div class="year_label">
+                                    <div class="text">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <div class="buttons">
+                                <div class="see_project">See project</div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="card_works">
+                    <div class="snapshot_portfolio">
+                    </div>
+                    <div class="left_block underline">
+                        <div class ="primary_text">
+                            <div class="project_title "> Multi-Post Stories</div>
+                            <div class="sub_header">
+                                <div class="client">
+                                    <div class="client_text ">Canopy</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <div class="text underline">Back End Dev</div>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year_label">
+                                    <div class="text underline">2015</div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class ="primary_text_1">A daily selection of 
+                            privately personalized reads; no accounts or 
+                            sign-ups required.
+                        </div>
+                        <div class="tags">
+                            <div class="tag_1 tag_html">html</div>
+                            <div class="tag_2 tag_css">css</div>
+                            <div class="tag_3 tag_javascript">javascript</div>
+                        </div>
+                        <div class="actions">
+                            <div class="buttons">
+                                <div class="see_project">See project</div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
```

Comment:
Similar to the above `Works Section`, this is a section and should be encapsulated on a `<section>` tag. So kindly change 
`<div>` to `<section>`